src/java/org/apache/cassandra/schema/SystemKeyspaceConstants.java

```diff
@@ -19,7 +19,7 @@
 
 import com.google.common.collect.ImmutableList;
 
-public final class SystemKeyspaceCts
+public final class SystemKeyspaceConstants
```

Comment:
Should we make this `SchemaKeyspaceConstants`, or are we planning on putting other system keyspace stuff in here?

Reply:
Honestly, might as well name it `SchemaKeyspaceTables` initially?

Let's also throw a private constructor in here so we can't instantiate.